apps/system/utils/netcmd.c

```diff
@@ -206,6 +206,25 @@ int cmd_ifdown(int argc, char **argv)
 	return ret;
 }
 
+int cmd_netmon(int argc, char **argv)
+{
+	FAR char *intf = NULL;
+	int ret;
+
+	if (argc > 2) {
+		printf("Bad input\n");
+		return OK;
+	}
+	if (argc == 1) {
+		intf = "all\n";
+	} else {
```

Comment:
Because of line 214 conditional, this else is not possible to execute. There is no chance to run this else.

Reply:
Please ignore my comment. As we talk on offline, I mis-understood.

Reply:
Thanks for the comment.